### ISSUE TITLE
About the status "Currently not working, fix coming soon..."

### ISSUE THREAD

**Issue by TheRealMephisto** [gamersi/chefkoch-api#2]
Hey,
you write in the "About" section of this repository:

"(CURRENTLY NOT WORKING, FIX COMING SOON!!) "

Why is it not working and when will the fix be here?
Could you use some help?

Kind Regards

**Comment by gamersi** [comment]
Hello, thanks for noticing. I tried it a few weeks ago and it was not working. I figured out that it is that way because Chefkoch has changed tags and classes and thats why it finally broke, because as you might know webscraping relies heavily on that. I will have to rewrite some stuff anyways.
I have not had enough time to fix this until now. If you want to feel free to make a pull request, otherwise I will fix it in the coming days
